src/go/rpk/pkg/cli/cmd/wasm/deploy.go

```diff
@@ -0,0 +1,99 @@
+package wasm
+
+import (
+	"encoding/binary"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"vectorized/pkg/kafka"
+	"vectorized/pkg/utils"
+
+	"github.com/Shopify/sarama"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewDeployCommand(
+	fs afero.Fs, createProducer func(bool, int32) (sarama.SyncProducer, error),
+) *cobra.Command {
+	var (
+		description string
```

Comment:
if it's one var, why have a list declaration?

Reply:
Yes, I forgot to remove this format after delete another variables